entities/GameState.js

```diff
@@ -11,6 +11,7 @@ class GameState {
   players = [];
   _allPlayerActions = {};
   _currentPlayerActions = {};
+  speedingCounter = 0;
```

Comment:
did I mistakenly remove this?

Reply:
I don't think we had put it in at all, actually

Reply:
magic